lib/launcher.rb

```diff
@@ -106,6 +106,7 @@ def https_app(https_port, plugins = https_plugins)
         :SSLOptions => ssl_options,
         :SSLCiphers => CIPHERS - Proxy::SETTINGS.ssl_disabled_ciphers,
         :daemonize => false,
+        :AccessLog => [],
```

Comment:
It took me a moment to understand how this was to unify it. Would it make sense to have a `base_app_settings` that returns all common options between HTTP and HTTPS? Things like `Logger`, `ServerSoftware` etc should be there and then merged with HTTP(S) specific options (`Port`, SSL settings).

Reply:
I wanted to make the changes as small as possible, but sure, done